azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -437,12 +451,12 @@ public LogData readFlowLogs(final int execId, final int startByte, final int len
       if (logFile.exists()) {
         return FileIOUtils.readUtf8File(logFile, startByte, length);
       } else {
-        logger.warn("Flow log file does not exist for flow execId: {}", execId);
+        logger.warn(String.format("Flow log file does not exist for flow execId: %d", execId));
         throw new ExecutorManagerException("Flow log file does not exist.");
       }
     } catch (final IOException e) {
-      logger.warn("IOException while trying to read flow log file for flow execId: {}",
-          execId);
+      logger.warn(String.format("IOException while trying to read flow log file for flow execId: %d",
```

Comment:
logger.error
For every place where logger.warn is there in catch.

Reply:
These were just modified to use log4j for the purpose of this PR. @sshardool did this part, would like to get his input before I make the change.
Perhaps, can be done in separate PR?

Reply:
I think this is the comment you intended to reply :)